GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -837,6 +837,74 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
                         }
                     }
                     break;
+                case (int)ArcDPSEnums.TrashID.VoidAmalgamate:
+                    EffectGUIDEvent voidPool = log.CombatData.GetEffectGUIDEvent(EffectGUIDs.HarvestTempleVoidPoolCM);
+                    if (voidPool != null)
+                    {
+                        var poolEffects = log.CombatData.GetEffectEventsByEffectID(voidPool.ContentID).Where(x => x.Src.Agent == target.AgentItem.Agent).ToList();
+                        // sort pool effects by time so that we can grow each effect
+                        poolEffects.Sort((a, b) => a.Time.CompareTo(b.Time));
```

Comment:
There should be a Any() check here, the .Last() below could throw an exception if the list is empty